CMakeLists.txt

```diff
@@ -83,6 +83,12 @@ if(FORCE_PORTABLE_INSTALL)
   set(CMAKE_INSTALL_LIBDIR ".")
   set(CMAKE_INSTALL_DATADIR ".")
   set(CMAKE_INSTALL_DOCDIR ".")
+
+  set(D3_DATADIR "${CMAKE_INSTALL_DATADIR}")
+else()
+  set(CMAKE_INSTALL_DATADIR "${CMAKE_INSTALL_DATADIR}/Descent3")
+  # On system install files will go into /usr/share/Descent3
+  set(D3_DATADIR "${CMAKE_INSTALL_FULL_DATADIR}/Descent3")
```

Comment:
I feel like we're mixing 2 logics here that should not be mixed. CMake install variables control how the package is generated from build files in the end. Choosing one or another location for installation should be up to the game packager, and should not condition build-time variables such as `D3_DATADIR`. I want to be able to install to prepare for packaging for instance in a subdir of the build directory, and avoid impacting default system data directory. The `D3_DATADIR` should be set as separate CMake variable.

Reply:
This is extension of our `FORCE_PORTABLE_INSTALL` compile option. If `FORCE_PORTABLE_INSTALL` is ON, we enforces `D3_DATADIR` to use current path, that way how we, developers, can use local builds. Distribution packagers should use `-DFORCE_PORTABLE_INSTALL=OFF`, and all data files will be installed and searched through `"${CMAKE_INSTALL_DATADIR}/Descent3"`. `CMAKE_INSTALL_DATADIR` can be redefined by packager into desired path (`/usr/share/Descent3` on most distributions).

Reply:
ok that makes more sense, thanks for the explanation